research/src/pages/focusgroup/focusgroup.explainer.mdx

```diff
@@ -480,14 +482,14 @@ linear group as described previously.
 Example 10:
 
 ```html
-<vertical-menu focusgroup="vertical wrap">
-  <menu-item tabindex="-1">Action 1</menu-item>
-  <menu-item tabindex="0">Action 2</menu-item>
-  <menu-group focusgroup="vertical extend">
-    <menu-item tabindex="-1">Sub-Action 3</menu-item>
-    <menu-item tabindex="-1">Sub-Action 4</menu-item>
+<vertical-menu role=menu focusgroup="vertical wrap">
+  <menu-item role=menuitem tabindex="-1">Action 1</menu-item>
+  <menu-item role=menuitem tabindex="0">Action 2</menu-item>
+  <menu-group role=none focusgroup="vertical extend">
```

Comment:
Shouldn't the role be menu here? 

> Submenus, also known as pop-up menus, also have the role menu.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/roles/menu_role#description

Reply:
If this was a submenu, that would be correct, but in fact, the "menu-group" is just acting like a generic div for some grouping purpose. The example is highlighting that the vertical extend is just making a continuation of the existing vertical menu (which is assumed to be structurally inline with the other menu options; who knows what the visual presentation is doing.)

Maybe just make it a 'div' instead?

Reply:
Will keep as `menu-group` for now.